mlxtend/plotting/corr_circle_pca.py

```diff
@@ -0,0 +1,97 @@
+# mlxtend Machine Learning Library Extensions
+#
+# A function for plotting a PCA correlation circle
+# File Author: Gabriel Azevedo Ferreira <az.fe.gabriel@gmail.com>
+
+from sklearn.decomposition import PCA
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from adjustText import adjust_text
+
+
+def create_correlation_table(A, B, names_cols_A, names_cols_B):
+    """
+    Compute the correlation coefficients and return as a DataFrame.
+
+    A and B: 2d array like.
+        The columns represent the different variables and the rows are
+        the samples of thos variables
+    names_cols_A/B : name to be added to the final pandas table
+
+    return: pandas DataFrame with the corelations.Columns and Indexes
+    represent the different variables of A and B (respectvely)
+    """
+    corrs = np.corrcoef(np.transpose(A), np.transpose(B)
+                        )[len(names_cols_A):, :len(names_cols_A)]
+    df_corrs = pd.DataFrame(corrs,
+                            columns=names_cols_A, index=names_cols_B)
+    return df_corrs
+
+
+def plot_pca_correlation_graph(X, variables_names, dimensions=(1, 2),
+                               figure_axis_size=6, X_pca=None):
+    """
+    Compute the PCA for X and plots the Correlation graph
+
+    X : 2d array like.
+        The columns represent the different variables and the rows are the
+         samples of thos variables
+    variables_names : array like
+        Name of the columns (the variables) of X
+    dimensions: tuple with two elements.
+        dimensions to be plot (x,y)
+    X_pca : optional. if not provided, compute PCA independently
+    figure_axis_size :
+         size of the final frame. The figure created is a square with length
+         and width equal to figure_axis_size.
+
+    returns:
```

Comment:
Also here, the `returns` should be formatted like the other docstrings so that the documentation will be formatted ok